cli/cmd/inject.go

```diff
@@ -26,6 +26,7 @@ const (
 	hostNetworkDesc                  = "pods do not use host networking"
 	sidecarDesc                      = "pods do not have a 3rd party proxy or initContainer already injected"
 	injectDisabledDesc               = "pods are not annotated to disable injection"
+	uncompatbleNodeOSDesc				 = "pods do not run on a windows node"
```

Comment:
`unsupportedOS = "pods cannot run on windows"`

Reply:
@vaniisgh You might also have fix the [formatting](https://github.com/linkerd/linkerd2/blob/master/BUILD.md#formatting) here and in a few other places so that the CI checks pass :)

Reply:
ya, I saw that :( . for some reason, my editor seems to show them differently! fixing that.